applications/StructuralMechanicsApplication/python_scripts/structural_response.py

```diff
@@ -191,15 +191,35 @@ class MassResponseFunction(ResponseFunctionBase):
 
     def __init__(self, identifier, response_settings, model):
         self.identifier = identifier
+
+        default_settings = Parameters("""{
+            "response_type"            : "mass",
```

Comment:
Are you sure this should have a default?
why not `SPECIFY_RESPONSE_TYPE`?

Reply:
because this is already the mass response.